.github/workflows/test.yml

```diff
@@ -30,17 +32,17 @@ jobs:
         path: ~/go/pkg/mod
         key: ${{ runner.os }}-${{ hashFiles('**/go.sum') }}
 
-    - name: Get dependencies
-      run: go get
+    - name: Vendor
+      run: go mod vendor
```

Comment:
Curious why we chose to go the vendoring route here?

Reply:
It has been faster for me to vendor and import vendored directory to buildkit rather than letting each container go get and rebuild from cache.  The go.hlb build will still force go-get, but when we move to `openllb/go.hlb` I was planning to move to vendor for those builds also.  The goal is to only resolve the deps once for each PR build.

Reply:
Using a mount for caching doesn't have the same effect?

Reply:
I have removed the vendor stuff, we can work this out later.